pkg/apis/eventing/v1beta1/eventtype_types.go

```diff
@@ -65,6 +65,11 @@ type EventTypeSpec struct {
 	// It may be a JSON schema, a protobuf schema, etc. It is optional.
 	// +optional
 	Schema *apis.URL `json:"schema,omitempty"`
+	// SchemaData allows the CloudEvents schema to be stored directly in the
+	// EventType. Content is dependent on the encoding. Optional attribute.
+	// The contents are not validated or manipulated by the system.
+	// +optional
+	SchemaData string `json:"schemaData,omitempty"`
```

Comment:
this is gonna get big. Is that an issue?

Reply:
Or maybe looking like this? 

https://github.com/n3wscott/discovery/blob/master/pkg/apis/discovery/v1alpha1/ducktype_types.go#L70

Reply:
that ends up being a JSON schema - not sure we want to place that restriction

put putting it as a string, I guess we are requiring it is text at a minimum

Reply:
re: size - I don't immediately have concerns about that, but maybe I'm missing something.